plugins/com.google.cloud.tools.eclipse.appengine.ui.test/pom.xml

```diff
@@ -18,6 +18,11 @@
       <plugin>
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-surefire-plugin</artifactId>
+        <configuration>
+          <useUIHarness>true</useUIHarness>
+          <useUIThread>true</useUIThread>
```

Comment:
FYI, this is true be default, but I think it's informative to have this line here.